### PR TITLE
Add test for unreachable tee_local value

### PR DESCRIPTION
**Pull request by binji**
See https://github.com/WebAssembly/wabt/issues/586.

### CODE REVIEW

File: test/core/unreached-invalid.wast [hunk 1]
```diff
@@ -623,3 +623,11 @@
   ))
   "type mismatch"
 )
+
+(assert_invalid
+  (module (func $tee-local-unreachable-value
+    (local i32)
+    tee_local 0 (unreachable))
```
Comment (rossberg-chromium) [inline test/core/unreached-invalid.wast]: Nit: parens for consistency :)

Comment (rossberg-chromium) [inline test/core/unreached-invalid.wast]: Actually, the bug in Wabt seems to be for `(unreachable) (tee_local 0)`, while this test is the other way round. Is that intentional?

Comment (binji) [inline test/core/unreached-invalid.wast]: Oops

**Comment by binji** [comment]
Travis seems to be wedged, maybe from the GH outage this morning? The other bot passed, so I'm assuming it is OK.

**Pull request merged**